api/src/pcapi/routes/backoffice/templates/accounts/get/general.html

```diff
@@ -131,6 +131,20 @@ <h6 class="card-subtitle text-muted">
           <div class="d-flex flex-column ms-auto align-items-end justify-content-end mb-1">
             {% if suspension_form %}
               {% if user.isActive %}
+                <div>
+                  {% if has_permission("EXTRACT_PUBLIC_ACCOUNT") and beneficiary_extract_tool_features %}
+                    {% if not has_gdpr_extract %}
+                      <form action="{{ url_for('.create_extract_user_gdpr_data', user_id=user.id) }}"
+                            name="{{ url_for('.create_extract_user_gdpr_data', user_id=user.id) | action_to_name }}"
+                            method="post">
+                        {{ extract_user_form.csrf_token }}
+                        <button class="btn btn-outline-primary mt-2 lead fw-bold">Extraire les données</button>
+                      </form>
+                        {% else %}
+                        <button class="btn btn-outline-primary mt-2 lead fw-bold" disabled>Extraire les données</button>
+                    {% endif %}
```

Comment:
[detail] attention a l'indentation du else

```suggestion
                    {% if not has_gdpr_extract %}
                      <form action="{{ url_for('.create_extract_user_gdpr_data', user_id=user.id) }}"
                            name="{{ url_for('.create_extract_user_gdpr_data', user_id=user.id) | action_to_name }}"
                            method="post">
                        {{ extract_user_form.csrf_token }}
                        <button class="btn btn-outline-primary mt-2 lead fw-bold">Extraire les données</button>
                      </form>
                    {% else %}
                        <button class="btn btn-outline-primary mt-2 lead fw-bold" disabled>Extraire les données</button>
                    {% endif %}
```